<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Res_session</class>
 <widget class="QWidget" name="Res_session">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Res_session</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #edebec</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>511</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>About</string>
    </property>
   </widget>
   <widget class="QComboBox" name="workout_combo">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 11px;
border-width: 1px;
background-color: #FFFFFF;
</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <item>
     <property name="text">
      <string>Legs +calves</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Trainer</string>
    </property>
   </widget>
   <widget class="QComboBox" name="slot_combo">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 11px;
border-width: 1px;
background-color: #FFFFFF;
</string>
    </property>
    <item>
     <property name="text">
      <string>8:00 - 10:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10:00 - 12:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14:00 - 16:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16:00 - 18:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18:00 - 20:00</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Slot</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Reserve session </string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Day</string>
    </property>
   </widget>
   <widget class="QComboBox" name="trainer_combo">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 11px;
border-width: 1px;
background-color: #FFFFFF;
</string>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="confirm_button">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>211</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset ; 
border-radius: 11px;
border-color:  #7D43E2;
border-width: 1px;
background-color: #7D43E2;
color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="date_chosen">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 11px;
border-width: 1px;
background-color: #FFFFFF;
color: #000000;
padding: 5px;
</string>
    </property>
    <property name="maximumDate">
     <date>
      <year>2022</year>
      <month>12</month>
      <day>31</day>
     </date>
    </property>
    <property name="minimumDate">
     <date>
      <year>2022</year>
      <month>1</month>
      <day>31</day>
     </date>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::DaySection</enum>
    </property>
    <property name="displayFormat">
     <string>dd-MM-yyyy</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>651</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(C:/Users/youssef albali/OneDrive/Desktop/SEM4/CP4/0f2c9051-gr12-repo/Project/GymAppQt/purple_background.png)</string>
   </property>
  </widget>
  <zorder>widget_2</zorder>
  <zorder>widget</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
